test/e2e/cluster_upgrade.go

```diff
@@ -119,83 +243,92 @@ var _ = Describe("Skipped", func() {
 
 		Describe("kube-push", func() {
 			It("of master should maintain responsive services", func() {
+				unsupported := []string{"gke"}
```

Comment:
Should we do this the other way around? If you list the providers that are supported I think this is easier to reason about (and for other folks to configure in their CI systems).